tests/idoverrideuser/test_idoverrideuser.yml

```diff
@@ -0,0 +1,509 @@
+
+  - name: Ensure test user test_user is present in idview test_idview with description, again
+    ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      description: "test_user description"
+    register: result
+    failed_when: result.changed or result.failed
+
+  - name: Ensure test user test_user is present in idview test_idview without description
+    ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      description: ""
+    register: result
+    failed_when: not result.changed or result.failed
+
+  - name: Ensure test user test_user is present in idview test_idview without description, again
+    ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      description: ""
+    register: result
+    failed_when: result.changed or result.failed
+
+  # name
+
+  - name: Ensure test user test_user is present in idview test_idview with internal name test_123_user
+    ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      name: test_123_user
+    register: result
+    failed_when: not result.changed or result.failed
+
+  - name: Ensure test user test_user is present in idview test_idview with internal name test_123_user, again
+    ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      name: test_123_user
+    register: result
+    failed_when: result.changed or result.failed
+
+  - name: Ensure test user test_user is present in idview test_idview without internal name
+    ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      name: ""
+    register: result
+    failed_when: not result.changed or result.failed
+
+  - name: Ensure test user test_user is present in idview test_idview without internal name, again
+    ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      name: ""
+    register: result
+    failed_when: result.changed or result.failed
+
+  # uid
+
+  - name: Ensure test user test_user is present in idview test_idview with uid 20001
+    ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      uid: 20001
+    register: result
+    failed_when: not result.changed or result.failed
+
+  - name: Ensure test user test_user is present in idview test_idview with uid 20001, again
+    ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      uid: 20001
+    register: result
+    failed_when: result.changed or result.failed
+
+  - name: Ensure test user test_user is present in idview test_idview without uid
+    ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      uid: ""
+    register: result
+    failed_when: not result.changed or result.failed
+
+  - name: Ensure test user test_user is present in idview test_idview without uid, again
+    ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      uid: ""
+    register: result
+    failed_when: result.changed or result.failed
+
+  # gecos
```

Comment:
We had issues before with GECOS not supporting non-ascii-7 characters.

Do you think it would be nice to have some test using unicode characters, like Ç, œ or ß ? The test we current have (ipauser) uses the Cyrillic representation for the name Yuri Gagarin.

Reply:
Yes, good point.

Reply:
Fixed.